### DISCUSSION TITLE
useCookie and sameSite

### DISCUSSION THREAD

**Discussion by Khal-ID** [nuxt/nuxt#16633] [Questions]
Hello, I was wondering if somebody could provide an example to implement sameSite with useCookie in Nuxtjs3. I'm new to Nuxt, I just wanna get rid of Google Analytics warning about sameSite and I have no clue how to do it. 

Any help would be appreciated.

TIA.

**Comment by fathi91** [comment]
Hi,
useCookie is not for setting a responce cookie it's for getting them and it's deprecated you can use getCookie instead,
for setting cookies you can use setCookie or the responce object, here is a working example from my project:

in `server/api/logou.ts` 
```javascript
export default defineEventHandler((e) => {                                                     // headers 
    e.res.writeHead(302 /* status code */, { Location: '/login', 'Set-Cookie': 'Authentication=; Max-Age=0; Path=/; HttpOnly; SameSite=Strict' })
    e.res.end()
})
```
